src/logger.ts

```diff
@@ -70,7 +70,8 @@ const defaults = {
         'responsePayload.token',
         'smsMessage.Body',
         'smsMessage.payload.body',
-        'token'
+        'token',
+        'res.headers.armor-session-token'
```

Comment:
It looks like this will have to go as
```suggestion
        'res.headers["armor-session-token"]'
```
see: https://github.com/pinojs/pino/issues/480

Reply:
Got it, fixed, thanks!